config/locales/en.yml

```diff
@@ -406,9 +406,9 @@ en:
           title: 'Her Majesty Queen Elizabeth II'
           href: /government/topical-events/her-majesty-queen-elizabeth-ii
           image_src: homepage/young-queen.jpg
-        - text: Prime Minister Liz Truss's statement on the death of Her Majesty The Queen.
-          title: Prime Minister Liz Truss's statement on the death of Her Majesty The Queen
-          href: /government/speeches/prime-ministers-statement-on-the-death-of-her-majesty-queen-elizabeth-ii
+        - text: Information about how to attend the Lying-in-State at the Palace of Westminster
+          title: Her Majesty The Queen's Lying-in-State
```

Comment:
Should use one of these: `’`, I can't think of the name, a fancy apostrophe.

Reply:
A curly quote!

Reply:
👍